src/arche/tools/api.py

```diff
@@ -144,7 +144,9 @@ def get_items_with_pool(
         A numpy array of items
     """
     active_connections_limit = 10
-    processes_count = min(max(helpers.cpus_count(), workers), active_connections_limit)
+    processes_count: int = int(
+        min(max(helpers.cpus_count(), workers), active_connections_limit) or 0
+    )
```

Comment:
That blows my mind - why casting and or 0?
`cpus_count` returns int
workers and active_connections_limit are both int.

Reply:
if I don't cast there's this error:
`src/arche/tools/api.py:147: error: Incompatible types in assignment (expression has type "Optional[int]", variable has type "int")`

if I change the type of `processes_count` to `Optional[int]`, I get this:
`src/arche/tools/api.py:148: error: Unsupported operand types for / ("int" and "None")
src/arche/tools/api.py:148: note: Right operand is of type "Optional[int]"`

if I use cast and don't add the `or 0`:
`src/arche/tools/api.py:147: error: Argument 1 to "int" has incompatible type "Optional[int]"; expected "Union[str, bytes, SupportsInt]"`

Reply:
```suggestion
        processes_count: int = min(
        max(helpers.cpus_count() or 0, workers), active_connections_limit
    )
```

Reply:
```
    processes_count: int = min(
        max(helpers.cpus_count() or 0, workers), active_connections_limit
    )
```